source/common/network/utility.cc

```diff
@@ -438,6 +439,9 @@ void Utility::addressToProtobufAddress(const Address::Instance& address,
     auto* socket_address = proto_address.mutable_socket_address();
     socket_address->set_address(address.ip()->addressAsString());
     socket_address->set_port_value(address.ip()->port());
+    if (address.networkNamespace().has_value()) {
```

Comment:
Copying out whatever we have here lgtm. Broadly, do we have special handling for empty networkspace when this was introduced?

Reply:
I don't think we have special handling for empty net namespaces, basically empty value here means the default network namespace (host network) is used for the sockets

Reply:
@botengyao friendly ping :-)